src/main/java/org/kohsuke/github/GHEventPayload.java

```diff
@@ -147,6 +489,11 @@ public String getBefore() {
             return before;
         }
 
+        @JsonSetter // alias
+        private void setAfter(String after) {
```

Comment:
It's `private` just using this as an alias field name... the documented sample event payload says "after" the documented event says "head" and my tests say "head" but the alias is here as a "just in case"